ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPool.java

```diff
@@ -86,23 +80,33 @@ public TransactionPool(
       final TransactionBroadcaster transactionBroadcaster,
       final EthContext ethContext,
       final MiningParameters miningParameters,
-      final MetricsSystem metricsSystem,
+      final TransactionPoolMetrics metrics,
       final TransactionPoolConfiguration configuration) {
     this.pendingTransactions = pendingTransactions;
     this.protocolSchedule = protocolSchedule;
     this.protocolContext = protocolContext;
     this.transactionBroadcaster = transactionBroadcaster;
     this.miningParameters = miningParameters;
+    this.metrics = metrics;
     this.configuration = configuration;
-
-    duplicateTransactionCounter =
-        metricsSystem.createLabelledCounter(
-            BesuMetricCategory.TRANSACTION_POOL,
-            "transactions_duplicates_total",
-            "Total number of duplicate transactions received",
-            "source");
-
     ethContext.getEthPeers().subscribeConnect(this::handleConnect);
+    initLogForReplay();
```

Comment:
The log for replay is separated from the normal one, for 2 main reasons: is it possible to enable/disable it without affecting the normal log, and because the output format is meant to be machine readable since a CSV is produced in a specific file.
More details in the javadoc of this class
https://github.com/fab-10/besu/blob/layered-txpool/ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/transactions/layered/ReplayTest.java#L73